src/runtime_src/core/pcie/tools/xbmgmt/cmd_flash.cpp

```diff
@@ -660,6 +665,78 @@ static int update(int argc, char *argv[])
     if (shell.empty() && !id.empty())
         return -EINVAL;
 
+    if(!xclbin.empty())
+    {
```

Comment:
keep curly brace on same line as conditional

Reply:
corrected the as per coding style